packages/frontend/src/features/sqlRunner/components/visualizations/BarChart.tsx

```diff
@@ -0,0 +1,34 @@
+import EChartsReact from 'echarts-for-react';
+import { type EChartsReactProps } from 'echarts-for-react/lib/types';
+import { memo, type FC } from 'react';
+import { type useSqlQueryRun } from '../../hooks/useSqlQueryRun';
+import { useAppSelector } from '../../store/hooks';
+import { useBarChartDataTransformer } from '../../transformers/useBarChartDataTransformer';
+
+type BarChartProps = Omit<EChartsReactProps, 'option'> & {
+    data: NonNullable<ReturnType<typeof useSqlQueryRun>['data']>;
+};
+
+const BarChart: FC<BarChartProps> = memo(({ data, className, ...rest }) => {
```

Comment:
```suggestion
const BarChart: FC<BarChartProps> = memo(({ data }) => {
```

and we can update the props above ⬆️